--- a/arch/arm/cpu/armv8/gxb/firmware/scp_task/Makefile 	2016-08-04 16:39:21.000000000 +0800
+++ b/arch/arm/cpu/armv8/gxb/firmware/scp_task/Makefile	2016-10-15 00:47:44.000000000 +0800
@@ -6,7 +6,7 @@
 include $(buildtree)/.config
 
 # Select ARMv7-m bare-metal toolchain
-CROSS_COMPILE=arm-none-eabi-
+CROSS_COMPILE=arm-eabi-
 ASM=$(CROSS_COMPILE)as
 CC=$(CROSS_COMPILE)gcc
 CPP=$(CROSS_COMPILE)cpp
@@ -20,7 +20,7 @@
             -fno-strict-aliasing -fno-common -fomit-frame-pointer         \
             -Werror-implicit-function-declaration -Wno-format-security    \
             -fno-delete-null-pointer-checks -Wdeclaration-after-statement \
-            -Wno-pointer-sign -fno-strict-overflow -fconserve-stack
+            -Wno-pointer-sign -fno-strict-overflow -fconserve-stack -Wno-unused-but-set-variable
 CFLAGS_DEBUG=-g
 
 # CPU specific compilation flags
